src/main/flight/pid.c

```diff
@@ -253,21 +253,77 @@ void pidInitConfig(const pidProfile_t *pidProfile) {
     relaxFactor = 1.0f / (pidProfile->setpointRelaxRatio / 100.0f);
     levelGain = pidProfile->P8[PIDLEVEL] / 10.0f;
     horizonGain = pidProfile->I8[PIDLEVEL] / 10.0f;
-    horizonTransition = 100.0f / pidProfile->D8[PIDLEVEL];
+    horizonTransition = (float)pidProfile->D8[PIDLEVEL];
```

Comment:
Sure, I'll look at doing it with more pre-loaded variables.  Just be aware that it's going to need 3 or 4 static floats, with only some of them used at run time (depending on the config at the time).

--ET